pages/heart.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: Holistic Enterprise Accessibility Review Technique (HEART)
+description: Ensuring Government Digital Services Accessibility
+permalink: /heart
+
+---
+
+![Image with the 5 phases of the HEART process](https://lh5.googleusercontent.com/7Cl86hJzMPTeYd8r312ACmsxVB0TOO0bMpoQYil2ooYPEnaFwTZ_yTc-TJ2W_ZxYOWwhd8MsXUCYH1GP5umfUmWaj_7gZut3t9mFTGaeI3L_iKMFk7tbVdoLfY6_mxV4lCtDKTZ5)
+
+## About
+
+Holistic Enterprise Accessibility Review Technique (HEART) is a proactive, interactive process to ensure government digital services are accessible to everyone.
+
+- [2 Pager](https://docs.google.com/document/d/1rZxWteDvRIJDuc0AN381oDlZhtyZQQZzm5GD3RXbKmY/edit#)
```

Comment:
Sorry about the complaints.

Reply:
NP..  Happy to have your feedback. I can see that it would be useful to adjust this to highlight that it is going to a GoogleDoc, but not sure what needs to be done here.

Reply:
I think the only change is replacing the `-` with a `*`